### ISSUE TITLE
Add web page listing OHW22 projects

### ISSUE THREAD

**Issue by emiliom** [oceanhackweek/oceanhackweek.github.io#228]
We haven't created a page listing all the projects that were carried out in OHW22. For example, a page [like this one from OHW21](https://oceanhackweek.org/ohw-resources/projects/projectlist/), which lists each project, the repo link, Youtube recording link, and a brief project summary.

At the root level of our OHW Google Drive there's a Google Sheet document ("ohw_proj_summary") that lists projects from each year, including OHW22. @leewujung was the last one to edit it. I assume that will be our starting point for the OHW22 projects page.

https://oceanhackweek.org/about/pasthackweeks.html is a single go-to page to all the OHW events. Once we have the OHW22 projects page, let's add an entry and link in that Past OHW Events page.

**Comment by leewujung** [comment]
Yep, I started that ohw_proj_summary  google sheet to tally the type of projects we have in 4 crude categories for making a summary slide. I think I got all projects counted in for 2022, but would be good for someone to double check.
